### ISSUE TITLE
Danish WMS server from SDFI does not work anymore

### ISSUE THREAD

**Issue by wonder-sk** [MerginMaps/input#2572]
When using a particular WMS server (`https://api.dataforsyningen.dk/orto_foraar_DAF`), the initial GetCapabilities request fails to finish properly.

- Regression in Input >= 2.0
- This can be reproduced also outside of Input / QGIS, just by using Qt's networking classes.
- Only Qt6 is affected, it works fine on Qt5 build of the same code
- It works fine with other HTTP(S) URLs, only this server seems to be affected

**Comment by wonder-sk** [comment]
Apparently since Qt6, HTTP/2 protocol is enabled by default (in Qt5 it was not enabled by default): https://codereview.qt-project.org/c/qt/qtbase/+/309558

Disabling HTTP/2 makes things work - an attribute needs to be set on network requests:
```cpp
request.setAttribute(QNetworkRequest::Http2AllowedAttribute, false);
```

Upstream Qt bug: https://bugreports.qt.io/browse/QTBUG-111417

**Comment by geomenke** [comment]
Thanks for this. When will this appear in the app? I have not seen any updates for the app and the problem is still there on my version of the app v 2.0.2 on android.

**Comment by tomasMizera** [comment]
Hi @geomenke, the fix is scheduled for v2.1.0 which will be released in the upcoming days (later this week or early next week)
